plugin_tests/client/mockVGL.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable camelcase */
+/* eslint-disable underscore/prefer-constant */
```

Comment:
but we could do it as another PR since you will need to bump up the version of geojs